dataland-external-storage/src/main/kotlin/org/dataland/datalandexternalstorage/utils/EurodatDataStoreUtils.kt

```diff
@@ -0,0 +1,42 @@
+package org.dataland.datalandexternalstorage.utils
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+/**
+ * Enables a centralized generation of log messages for all Dataland backend operations.
+ */
+
+@Component("EurodatDataStoreUtils")
+object EurodatDataStoreUtils {
+    private const val maxRetriesConnectingToEurodat = 8
+    private const val millisecondsBetweenRetriesConnectingToEurodat = 15000
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    /**
+     * This method will rerun a given method if an exception is thrown while running it
+     * @param inputMethod to specify in the logs which method should be rerun
+     */
```

Comment:
Vllt noch dazu schreiben, dass es nur für eine begrenzte Anzahl an retries versuchen wird, es zu rerunnen.

Und ich glaube der param für "block" fehlt noch.

Reply:
Ergänzt

Reply:
return fehlt noch.  vllt einfach ein Satz, dass der return value der Funktion (also quasi T) einfach nach außen returnt wird

Reply:
Ergänzt

Reply:
- Die Beschreibung hinter desciptionOfOperation ist nicht ganz richtig
- Die logs im error Fall werden komisch aussehen wegen "trying to $descriptionOfOperation".  Das "trying to" ist problematisch

Reply:
Der doc der Klasse ist falsch. ("nables a centralized generation of log messages for all Dataland backend operations.")